packages/eslint-config-kibana/jest.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
       },
 
       rules: {
-        'jest/no-disabled-tests': 'error',
```

Comment:
We do skip tests from time to time, so we can't "error" on them.

Reply:
But that's what `/* eslint-disable-next-line jest/no-disabled-tests */` is for :)

Reply:
This rule has definitely saved my butt more than a couple times.

Reply:
:+1: Okey, I agree. Fixing now

Reply:
Oh, sorry, you have to use a single line comment

```
// eslint-disable-next-line jest/no-disabled-tests
```

Reply:
Thinking more about this, as we don't have this specified for Mocha either I think we should just merge this as-is.

Reply:
🤷‍♂️ very well